.travis.yml

```diff
@@ -17,7 +17,6 @@ addons:
 
 cache:
   directories:
-  - node_modules
```

Comment:
@AdamWard1995 why?  This does not follow the practice of any of our other repos /cc @juwara0